listenbrainz/webserver/templates/index/playlists-management.html

```diff
@@ -0,0 +1,34 @@
+{%- extends 'base.html' -%}
+{%- block title -%}Playlists management - ListenBrainz{%- endblock -%}
+{%- block content -%}
+  <h2 class="page-title">Playlists managementg </h2>
+  {# Playlists management information #}
+  <h3>Synchronizing Playlists</h3>
+  <p>
+    There are many ways to sync your own playlists to ListenBrainz:
+  </p>
+  <h4>Music players</h4>
+  <ul>
+    <li><em><a href="https://www.foobar2000.org/">Foobar2000</a></em>, full-fledged audio player for Windows: <a href="https://github.com/regorxxx/Playlist-Manager-SMP"><code>Playlist-Manager-SMP</code></a></li>
+  </ul>
+
+  <h3>Synchronizing via third-party music services</h3>
+  <p>
+    <a href={{ url_for('profile.music_services_details') }}>Connect your ListenBrainz account</a> and import/export your playlists to some third-party services.
+  </p>
+  <ul>
+    <li><em><a href="https://open.spotify.com/">Spotify</a></em></li>
+  </ul>
+  <p>
+    
+  </p>
+
+  <h3>For advanced users</h3>
+  <p>
+    Developers are able to import/export and edit their playlists on Listenbrainz using the Listenbrainz API. Information on how to do this
+    can be found in the <a href="https://listenbrainz.readthedocs.io">API docs</a>.
+  </p>
+  <p>
+	Note playlist importing requires track MBID identification for every item; exporting requires some kind of content resolution of tracks by track MBID, title and/or artist, following <a href="https://xspf.org/spec#34-content-resolver">JSPF or XSPF specs</a>.
```

Comment:
We can link to our tool we use for that purpose: https://labs.api.listenbrainz.org/mbid-mapping which can also be called as an API

Reply:
Thanks! Didn't knew about it... in my foobar plugin the use-case was sending a playlist to the server with tracks already tagged with 'MUSICBRAINZ_TRACKID' so it was not required. But I will make good use of that to cover sending playlists with non tagged files.

What I found most needed **was an api to link MBIDs to youtube urls.** i.e. if your export a JSPF playlist for local use, it would be great to have the **youtube link found on the ListenBrainz player as default location source.** Right now you need to use content resolution to find matches with local tracks, but for missing items, those tracks would be simply skipped (while there is software out there able to play youtube URLs). Foobar2000 is an example.

Reply:
In the ListenBrainz player we do content resolution on the spot too with the youtube search API.
But we are indeed working on better content resolution, initially to get Spotify IDs for a given recording MBID with a goal to extend it to other sources.
We want it for the same reasons: content resolution on the fly is a bit uneven, and some search APIs leave to be desired, so we want to serve content URLs along with the rest of the metadata.
But we're not quite there yet.